qutebrowser/misc/utilcmds.py

```diff
@@ -234,3 +234,17 @@ def repeat_command(win_id, count=None):
     cmd = runners.last_command[mode_manager.mode]
     commandrunner = runners.CommandRunner(win_id)
     commandrunner.run(cmd[0], count if count is not None else cmd[1])
+
+@cmdutils.register(debug=True,name='debug-log-capacity')
+def log_capacity(capacity: int):
+    """Choose number of lines for your log.
+
+    Args:
+       capacity: Number of lines for the log.       
+    """
+    if capacity < 0:
+        raise cmdexc.CommandError("Please choose log capacity above 0.")
```

Comment:
To be consistent with other error messages, I think something like *"Can't set a negative log capacity!"* would read better.